core/src/main/java/org/apache/calcite/plan/volcano/RelSubset.java

```diff
@@ -438,6 +439,16 @@ void propagateCostImprovements0(VolcanoPlanner planner, RelMetadataQuery mq,
     return list;
   }
 
+  public Stream<RelSubset> getSubsetsSatisfingThis() {
+    return set.subsets.stream()
+      .filter(s -> s.getTraitSet().satisfies(traitSet));
+  }
```

Comment:
should be non-public

Reply:
Can we add this as a public api? This one is very similar to getParentRels(), getRels(), except that it is looking for relationships of RelSubset rather than rels.

Reply:
satisfying?

Reply:
fixed